imagechecker_postprocessing.rb

```diff
@@ -18,24 +20,54 @@
 missing_jpg = File.join(Bkmkr::Paths.scripts_dir, "bookmaker_assets", "pdfmaker", "images", "generic", "missing.jpg")
 
 # ---------------------- METHODS
+
+## wrapping a Mcmlln::Tools method in a new method for this script; to return a result for json_logfile
+def getFilesinDir(path, logkey='')
+	files = Mcmlln::Tools.dirListFiles(path)
+	return files
+rescue => logstring
+	return []
+ensure
+  Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+## wrapping a Mcmlln::Tools method in a new method for this script; to return a result for json_logfile
+def readOutputHtml(logkey='')
+	filecontents = File.read(Bkmkr::Paths.outputtmp_html)
+	return filecontents
+rescue => logstring
+	return ''
+ensure
+  Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
```

Comment:
indentation seems a but inconsistent throughout all files. FYI you should be using 2 spaces as the indent, rather than 4 (you can customize your tab defaults in Sublime).